spec/features/proposal_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 feature Event do
   let!(:conference) { create(:conference) }
+  let!(:registration_period) { create(:registration_period, conference: conference, start_date: Date.current) }
```

Comment:
Project confirmation gets redirected to new registration page. We need conference to be open for registration for the redirect.

We won't need it, after we have solved #802